app/views/articles/index.html.erb

```diff
@@ -1,11 +1,13 @@
-<% if current_user != nil %>
-  <% if current_user.role.include?("Admin") || current_user.role.include?("Journalist") || current_user.role.include?("Moderator") %>
-    <a href="/articles/new">Create a new article</a>
+<div class="container">
+  <% if current_user != nil %>
+    <% if current_user.role.include?("Admin") || current_user.role.include?("Journalist") || current_user.role.include?("Moderator") %>
```

Comment:
It feels like such a long if-statement can be shorten and improved by either creating an array where every element will be a role such as admin or journalist etcetera. If the current_user role has x from this array it should be allowed to create an article. If later on more roles will be able to create articles you do not need to change the whole if-statement but instead just add the role to the array.

Reply:
Didn't think of that! Much easier to read now.